example/image-classification/common/fit.py

```diff
@@ -168,10 +168,16 @@ def fit(args, network, data_loader, **kwargs):
 
     # callbacks that run after each batch
     batch_end_callbacks = [mx.callback.Speedometer(args.batch_size, args.disp_batches)]
+    eval_end_callbacks = []
     if 'batch_end_callback' in kwargs:
         cbs = kwargs['batch_end_callback']
         batch_end_callbacks += cbs if isinstance(cbs, list) else [cbs]
 
+    # tensorboard logs
+    if "eval_end_callback" in kwargs:
+	cbs = kwargs['eval_end_callback']
+        eval_end_callbacks += cbs if isinstance(cbs, list) else [cbs]
```

Comment:
So stupid I am, I forgot `make` and `pip install -e .`. I tested and verified this PR. It's okay on current master branch.